smart_contracts/contract/src/contract_api/storage.rs

```diff
@@ -335,3 +335,72 @@ pub fn disable_contract_version(
 
     api_error::result_from(result)
 }
+
+/// Creates new [`URef`] that points to a context-local partition of global state.
+/// Optionally you can pass a [`str`] to automatically put this dictionary under named keys.
```

Comment:
This comment is incorrect; "context-local partition" is a misleading / opaque explanation, and the &str argument isn't optional.